internals/cli/ui/io.go

```diff
@@ -18,8 +18,8 @@ var (
 
 // IO is an interface to work with input/output.
 type IO interface {
-	Stdin() io.Reader
-	Stdout() io.Writer
+	Input() io.Reader
+	Output() io.Writer
 	Prompts() (io.Reader, io.Writer, error)
 	IsStdinPiped() bool
 	IsStdoutPiped() bool
```

Comment:
Now these need to be renamed as well, right?

Reply:
Good point. Done.